init.lua

```diff
@@ -1,53 +1,54 @@
 local M = {}
 
-function M:peek()
-    local child = Command("bat")
-        :args({
-            "--style", "plain",
-            "--color", "always",
-            tostring(self.file.url),
-        })
-        :stdout(Command.PIPED)
-        :stderr(Command.PIPED)
-        :spawn()
+function M:peek(job)
```

Comment:
The change of the indentation level makes it hard t find the real difference between the versions.

Is the main change the "self" to "job" transition?

Reply:
Yeah, the main changes are the transition from self to job and replacing ui.Paragraph with ui.Text. The indentation issue has been fixed.